### DISCUSSION TITLE
鉴于使用时出现问题的一点小思考（增加肉鸽效率）

### DISCUSSION THREAD

**Discussion by mengyiran934** [MaaAssistantArknights/MaaAssistantArknights#10736] [Ideas]
今早我晨跑回来时，发现挂机的肉鸽中有一个图可以让高台下地面，然后就出现了凯尔希阻挡小怪，整个游戏卡在那里许久，这种情况并不常见，但是日常中也有奶盾和精英怪互啄好久的场面，我就在想，有没有办法可以提高效率（刷经验）
一种方法是时刻检测，比如发现医疗阻挡后，周围补干员来补伤害。
当然了，直接把这个医疗撤掉最省事。
但是要判断有无阻挡的话，要不然就得模型训练（或者模版匹配），哪怕训练好了，也很占运存（对于老一点的电脑不是很友好，会崩）。
然后我想，是不是可以给每把肉鸽都加个限制，每一小关卡，十分钟过不去就直接放弃，开下一把，哪怕少拿点经验也比卡在这里浪费时间好。
本来想上手试验的，但是那块代码有点没看懂，哪怕看懂了，后续怎么打包成exe进行尝试也是个问题，所以就发了出来，希望各位大佬能代码实现，这样能让肉鸽这个模块效率更高一点。
当然了，可能代码中有这类的保护我看不出，毕竟我菜菜的，或者有更好的增效且降低时间复杂度的方法在下面提出，谢谢大家

**Comment by mengyiran934** [comment]
有点啰嗦，总结一下就是，我想肉鸽每个小关卡都加个限制，每一小关卡，十分钟过不去就直接放弃，开下一把，哪怕少拿点经验也比卡在这里浪费时间好，但我菜，改不出代码

**Comment by Daydreamer114** [comment]
十分钟过不去就放弃这个东西一直存在，没放弃的话应该是出bug了，可以当bug issue发
